pkg/cmd/root_test.go

```diff
@@ -0,0 +1,20 @@
+package cmd
+
+import (
+	"github.com/google/go-cmp/cmp"
+	"github.com/tektoncd/cli/pkg/test"
+	"testing"
+)
+
+func TestCommand_invalid(t *testing.T) {
+	p := &test.Params{}
+	pipelinerun := Root(p)
+	out, err := test.ExecuteCommand(pipelinerun)
+	if err == nil {
+		t.Errorf("No errors was defined. Output: %s", out)
+	}
+	expected := "Use one of available subcommands: completion [SHELL], help [command], pipeline, pipelinerun, task, taskrun, version"
```

Comment:
nit: we may not want to check the exact output as adding a new command means changing this test.